### ISSUE TITLE
[V2] API in V2 

### ISSUE THREAD

**Issue by Nath31570** [opencve/opencve#425]
## Expected Behavior

Hello, Using the API call "/api/cve/cve_id", we are normally able to retrieve detailed information about a specific CVE bulletin, including the impacted CPEs.

## Actual Behavior

However, I just did a test on V2 and I realize that what is returned is completely different in terms of schema but also content. We have no information, neither on the impacted CPEs, nor a link to a reference.

Is this normal? Have the API calls changed (apart from the addition of organizations / projects etc.)? Is a doc for the V2 API planned?

## Steps to Reproduce the Problem

  - Make an API call with "/api/cve/<cve_id>"
  - Check the output to see if the CVE information matches what is mentioned on the web interface (for example)

## Specifications

  - OpenCVE version: V2

## Screenshots (optional)

With API V2 : 

![image](https://github.com/user-attachments/assets/03e9a08e-b389-4bae-98d9-1e9d3f175337)

With API V1 : 

![image](https://github.com/user-attachments/assets/8a094a66-a6fe-43b8-8b26-151ddd8aadc3)
![image](https://github.com/user-attachments/assets/659655ed-aa39-4731-a74f-775f59f5a4df)
![image](https://github.com/user-attachments/assets/c3b68ddd-383e-4f91-82fe-3fd2298e1c5d)

## Comments (optional)

Apart from this issue, I take this opportunity to congratulate you for the work done on this project and thank you for keeping it open-source. V2 includes major changes and it is a success ;) !

**Comment by ncrocfer** [comment]
Hi @Nath31570,

Thank you very much for your kind words—it’s always great to receive feedback like this 👍 

To answer your question, yes, the API has changed significantly in v2, and we acknowledge that the documentation around these updates needs improvement. We apologize for any inconvenience this has caused, and we’re prioritizing this to ensure you have access to updated documentation as soon as possible—by this week.

Beyond documentation, we’re planning to expand the API’s capabilities in the coming weeks and months. If any essential data is missing, we’ll work to integrate it to make the API as useful as possible for our users.

Thanks again for your feedback, and please don’t hesitate to reach out with more suggestions!

**Comment by ncrocfer** [comment]
Here is the API documentation: https://docs.opencve.io/api/

**Comment by umutcancakann** [comment]
I would like to ask a new question about the new API. In v1 we were able to get subscribed products of an account. When I looked up on the v2 api there wasn't any endpoint. Are you planning to add it?

**Comment by ncrocfer** [comment]
@umutcancakann , yes API will be improved. That said I don't have any ETA to give you but it will be in next months.

**Comment by Nath31570** [comment]
Hi @ncrocfer,

I catching up on the news regarding this topics. Do you have any information to share? I'm sure you have a lot of things in your backlog and you don't have a specific ETA but I was just wondering if there has been any progress on this.

**Comment by ncrocfer** [comment]
For visitors who might come across this old issue: https://docs.opencve.io/api/
